tests/steps/deploy/k8s/test_chart_builder.py

```diff
@@ -80,6 +81,13 @@ def test_should_validate_against_crd_schema():
 
 @pytest.mark.parametrize('template',
                          ['deployment', 'service', 'serviceaccount', 'sealedsecrets', 'ingress-https-route'])
-def test_chart_roundtrip(template):
-    charts = _build_chart()
+def test_service_chart_roundtrip(template):
+    charts = _build_chart(ServiceChartBuilder())
     _roundtrip(template_path, template, charts)
+
+
+@pytest.mark.parametrize('a',
+                         ['configmap', 'cronjob', 'scheduledsparkapplication', 'sealedsecret', 'serviceaccount'])
+def test_job_chart_roundtrip(a):
+    charts = _build_chart(JobChartBuilder())
+    _roundtrip(job_template_path, a, charts)
```

Comment:
This test is still failing because of a key error. Probably related to how these yamls were generated, which I am unsure of if I did the correct thing